apps/condo/domains/banking/tasks/importBankTransactionsFrom1CClientBankExchange.js

```
@@ -104,6 +105,24 @@ const importBankTransactionsFrom1CClientBankExchange = async (taskId) => {
         organization: { id: organization.id },
         deletedAt: null,
     })
+    if (property) {
+        const adminContext = await context.createContext({ skipAccessControl: true })
+        const accountByProperty = await BankAccount.getAll(adminContext, {
+            organization: { id: organization.id },
+            property: { id: property.id },
+            deletedAt: null,
+        })
+        if (!isEmpty(accountByProperty)) {
+            await BankSyncTask.update(context, task.id, {
+                status: BANK_SYNC_TASK_STATUS.ERROR,
+                meta: {
+                    errorMessage: `Already have an account with the same Property { id: ${property.id} }.`,
+                },
+                ...DV_SENDER,
+            })
+            return
```

Maybe throw an error and log some error to the logger instead of just return?